### DISCUSSION TITLE
Portability and embedded fonts

### DISCUSSION THREAD

**Discussion by scorster** [musescore/MuseScore#19100] [Ideas]
Admirably MuseScore is dedicated to document portability—i.e. consistent display on MuseScore.com as well as in the MuseScore app.) But MuseScore enforces this by restricting scorists to using fonts either baked into MuseScore or common across OS platforms. My spidey sense says the solution is simple: allow any font; if in any circumstance that fontFace is an unavailable resource, then have MuseScore resort to a fallback font.  

.org discussions on the matter exist, but I'm wondering if there are any Github discussions about retaining document portability by:

•  referencing WebFonts. Works for HTML and CSS! 

• alternately—when faced with a non-musescore-native font—MuseScore could preserve portability by embedding a subset of the font (i.e. only the glyphs used in the score) just as PDF files do. 

I hope options like those presented are being consider. I want to use some fonts I've made, for musical symbols like tablature fret numbers, fretboard diagrams, capo images ... and we've heard from other users who need the same flexibly to use a wider range of fonts.
scorster

**Comment by stupidvegan1** [comment]
Hey Scorster, wondering the exact same thing. Really need custom tab numbers but cannot figure out the workaround. I know nothing about coding unfortunately.

**Comment by scorster** [comment]
.org discussions and requests regarding font portability and using tablature fonts other than those provided internally:

https://musescore.org/en/node/366634#comment-1251465

https://musescore.org/en/node/299448#comment-1046530
